package.json

```diff
@@ -11,14 +11,15 @@
   "author": "Nicholas C. Zakas",
   "main": "lib/index.js",
   "scripts": {
-    "lint": "eslint lib/ tests/",
-    "lint:fix": "eslint lib/ tests/ --fix",
+    "lint": "eslint lib/ tests/ tools/",
+    "lint:fix": "eslint lib/ tests/ tools/ --fix",
     "docs": "eslint-docs",
     "docs:check": "eslint-docs check",
     "format-no-write": "prettier-eslint lib/**/*.js tests/**/*.js --eslint-config-path=.eslintrc --eslint-ignore --prettier-ignore --eslint-path=node_modules/eslint --config=.prettierrc",
     "format": "yarn format-no-write --write",
     "format-check": "yarn format-no-write --list-different",
-    "test": "mocha tests --recursive --reporter=dot"
+    "test": "mocha tests --recursive --reporter=dot",
+    "update": "node tools/update-recommended.js"
```

Comment:
could you please rename this to `recommended:update` just so it's more descriptive?